test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -57,11 +57,11 @@
 
         (testing "should attempt a partial migration if there are migrations missing"
           (clear-db-for-testing!)
-          ;; We are using migration 13 here because it is isolated enough to be able
+          ;; We are using migration 28 here because it is isolated enough to be able
```

Comment:
Oh I think I see what you are all saying. I'm a bit vague on the understanding, but sounds like this test should be tested on 'sample' migrations that have nothing to do with PDB so our own work can't break this. This test isn't about testing our real migrations, rather its about testing the migration code in general - so 'real' data isn't needed I think.